pyproject.toml

```diff
@@ -1,6 +1,22 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-[tool.distutils.bdist_wheel]
-universal = true
+[project]
+name = "yt-dlp-sample-plugins"
+version = "2023.01.02"
+readme = "README.md"
+requires-python = ">=3.8"
+license = "MIT"
```

Comment:
```suggestion
license = "Unlicense"
```

Reply:
This is a discrepancy between wanting people to NOT use Unlicense by default, and us licensing the code as Unlicense... But I'll make it Unlicense for the time being

Reply:
```toml
license = {file = "LICENSE"}
```
why not let the plugin developer decide which license to use theirself?

Reply:
Fair